app/experimenter/experiments/forms.py

```diff
@@ -570,13 +585,50 @@ class ExperimentObjectivesForm(ChangeLogMixin, forms.ModelForm):
         widget=forms.Textarea(attrs={"class": "form-control", "rows": 20}),
     )
 
+    survey = forms.TypedChoiceField(
+        label=Experiment.SURVEY_REQUIRED_LABEL,
+        help_text=Experiment.SURVEY_HELP_TEXT,
+        choices=RADIO_OPTIONS,
+        widget=RadioWidgetCloser,
+        coerce=coerce_truthy,
+        empty_value=None,
+    )
+    survey_urls = forms.CharField(
+        required=False,
+        help_text=Experiment.SURVEY_HELP_TEXT,
+        label="Survey URLs",
+        widget=forms.Textarea(attrs={"class": "form-control", "rows": 1}),
+    )
+    survey_instructions = forms.CharField(
+        required=False,
+        label=Experiment.SURVEY_INSTRUCTIONS_LABEL,
+        help_text=Experiment.SURVEY_LAUNCH_INSTRUCTIONS_HELP_TEXT,
+        widget=forms.Textarea(attrs={"class": "form-control", "rows": 10}),
+    )
+
```

Comment:
survey itself is required for the form... because a user can say No/Yes... is this correct? URLs and instructions are not required.

Reply:
That seems correct. Per discussions in Slack, these fields are not "required" even if a user says "Yes" to a survey, because although we need them, we can ask for them manually later.